Code/GraphMol/Wrap/rdchem.cpp

```diff
@@ -59,7 +59,10 @@ BOOST_PYTHON_MODULE(rdchem) {
       &translate_value_error);
   python::register_exception_translator<RDKit::MolSanitizeException>(
       &rdSanitExceptionTranslator);
-
+#if INVARIANT_EXCEPTION_METHOD  
```

Comment:
This belongs in rdbase (i.e. RDBoost/Wrap.RDBase.cpp), not here. That way the translator will be registered even if rdchem isn't imported.

I know we have exception translators for ValueError and IndexError registered in multiple places too, but I don't think it's necessary anymore (or maybe ever) and I will put it on the "to be cleaned up" list.